test/EditableComponent.test.tsx

```diff
@@ -232,6 +236,24 @@ describe('EditableComponent ->', () => {
 
             expect(node).toBeNull();
         });
+
+        it('if aem grid column styles set, appliedCssClassNames should not override the grid styles', () => {
+            const EDIT_CONFIG = {
+                isEmpty: function() {
+                    return false;
+                },
+                emptyLabel: EMPTY_LABEL,
+                resourceType: COMPONENT_RESOURCE_TYPE
+            };
+
+            const EditableComponent: any = withEditable(ChildComponent, EDIT_CONFIG);
+
+            ReactDOM.render(<EditableComponent isInEditor={true} {...CQ_PROPS}/>, rootNode);
+
+            const node = rootNode.querySelector('.' + GRID_CLASS_NAME);
+
+            expect(node).not.toBeNull();
+        });
```

Comment:
I would not use any constant in the test. I would use string literals. That way it is more easy to see the testing intentions.

Also I would use a different expectation instead querying and comparing to null, like comparing the CSS class attribute itself, so it is more explicit the test, something like:

`.toHaveClass('xyz abc')`